proto/endorsement.proto

```diff
@@ -73,11 +73,17 @@ message VMTdx {
     // The amount of RAM in GiB provided to the VM. This is relevant to the
     // construction of the measured TDHOB page that includes memory regions'
     // resource attributes.
+    //
+    // To be deprecated: The TDHOB is no longer measured in newer Google
```

Comment:
When will it be deprecated? When the buggy vanadium is no longer in production?

Reply:
Correct. I can remove it when the buggy VMM is irrelevant to account for.